plugin/manager.go

```diff
@@ -138,17 +138,21 @@ func (pm *Manager) tmpDir() string {
 // HandleExitEvent is called when the executor receives the exit event
 // In the future we may change this, but for now all we care about is the exit event.
 func (pm *Manager) HandleExitEvent(id string) error {
+	logrus.WithField("id", id).Debug("HandleExitEvent")
 	p, err := pm.config.Store.GetV2Plugin(id)
 	if err != nil {
 		return err
 	}
 
-	os.RemoveAll(filepath.Join(pm.config.ExecRoot, id))
+	if err := os.RemoveAll(filepath.Join(pm.config.ExecRoot, id)); err != nil && !os.IsNotExist(err) {
+		logrus.WithError(err).WithField("plugin", id).Error("Could not remove plugin bundle dir")
```

Comment:
Lets pick one format for `WithField`. Either `WithField("plugin", ...)` like here or `WithField("id, ...")` like in `containerd.go`. I like the former

Reply:
Easier to grep in log files for the same field string.